### PR TITLE
Replace deprecated JSCS with eslint

### PR DESCRIPTION
**Pull request by varshard**
Refer to [issue#65](https://github.com/omise/omise-node/issues/65) 

Replace deprecated JSCS with eslint as JSCS is merged with eslint.
Existing JSCS rules also converted into eslint's equivalent rules.

### CODE REVIEW

File: lib/api.js [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
-var Promise     = require('bluebird');
+var https = require('https');
+var Promise = require('bluebird');
 
-var logger      = require('./logger');
-var pkgjson     = require('../package.json');
+var logger = require('./logger');
+var pkgjson = require('../package.json');
 
-var ApiError    = require('./errors/api-error.js');
```
Comment (turboza) [inline lib/api.js]: Can't we use this style? 😢

File: test/mocks/customers_list.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ nock('https://api.omise.co')
       email: 'john.doe@example.com',
       description: 'John Doe (id: 30)',
       created: '2015-02-03T05:28:15Z',
-      cards: [Object]
+      cards: [Object],
```
Comment (turboza) [inline test/mocks/customers_list.js]: just saw we have this in the code. hopefully the tests are still valid 😂

File: lib/api.js (hunk 1 above)
Comment (schumarn) [reply lib/api.js]: Actually good point. Can we adjust rule so we can align assignment expression? @varshard

Comment (varshard) [reply lib/api.js]: I saw that some assignments are aligned in some and not aligned in some.
I will try to preserve the alignment, then.

File: test/mocks/customers_list.js (hunk 2 above)
Comment (varshard) [reply test/mocks/customers_list.js]: The dangling comma in object literal is valid. It's up to our agreement.

Currently, the rule that force dangling comma is enabled by Google's eslint configuration, but we can override it.

File: lib/api.js (hunk 1 above)
Comment (schumarn) [reply lib/api.js]: Let's force align all of them then 😄

Comment (varshard) [reply lib/api.js]: Unfortunately that there are no such option. 😢

File: test/mocks/card_retrieve.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ var nock = require('nock');
 
 var chargeID = 'cust_test_4z2owmajzsb3c527wj7';
 var cardID = 'card_test_4z2owrdmvbygi7ah0fu';
-var url = '/customers/' + chargeID + '/cards/' + cardID;
+var url    = '/customers/' + chargeID + '/cards/' + cardID;
```
Comment (turboza) [inline test/mocks/card_retrieve.js]: need 1 more space for both lines above?

**Pull request merged**